docs/api-inference/_toctree.yml

```diff
@@ -30,5 +30,11 @@
       title: Text Generation
     - local: tasks/text_to_image
       title: Text to Image
+    - local: tasks/token_classification
```

Comment:
I'm wondering if we should add an extra layer to split by modality, but not needed I think?

Reply:
I wondered as well. For now I'm listing everything by alphabetical order. I think it's fine to keep it like this but if you are opinionated we can keep the same "per-modality split" as in hf.co/tasks.

Reply:
(I'm always using ctrl+F so I'm not of good advice for this :see_no_evil:)

Reply:
Yeah let's wait for final review from others!